_docs/setup/install-kubernetes.md

```diff
@@ -126,39 +130,50 @@ You can find out more about how to use these tools in [Collecting Metrics and Lo
 <figcaption>Grafana Istio Dashboard</figcaption></figure>
 
 #### Verifying the ServiceGraph service
 
-The ServiceGraph addon provides a textual (JSON) representation and a graphical visualization of the service interaction graph for the cluster. Like Grafana, you can access the servicegraph service using port-forwarding, service nodePort, or (if external load balancing is available) external IP. In this case the service name is `servicegraph` and the port to access is 8088:
+The ServiceGraph add-on provides a textual (JSON) representation and a graphical visualization of the
+service interaction graph for the cluster. Like Grafana, you can access the servicegraph service using
+port-forwarding, service nodePort, or, if external load balancing is available, external IP. In this
+example the service name is `servicegraph` and the port to access is `8088`:
 
 ```bash
 kubectl port-forward $(kubectl get pod -l app=servicegraph -o jsonpath='{.items[0].metadata.name}') 8088:8088 &
 ```
 
-The ServiceGraph service provides both a textual (JSON) representation (via `/graph`) and a graphical visualization (via `/dotviz`) of the underlying service graph. To view the graphical visualization (assuming that you have configured port forwarding as per the previous snippet), open your browser at: [http://localhost:8088/dotviz](http://localhost:8088/dotviz).
+The ServiceGraph service provides both a textual (JSON) representation (via `/graph`) and a graphical
+visualization (via `/dotviz`) of the underlying service graph. If you configured port forwarding using
+the above command, you can view the graphical visualization by opening your browser at [http://localhost:8088/dotviz](http://localhost:8088/dotviz).
 
-After running some services -- for example, after installing the [BookInfo]({{home}}/docs/samples/bookinfo.html)  sample application and generating some load on the application (e.g., executing `curl` requests in a `while` loop) -- the resulting service graph should look something like this:
+After you run some services, a service graph builds. For example, after installing the
+[BookInfo]({{home}}/docs/samples/bookinfo.html) sample application and generating some load on the
+application (e.g., executing `curl` requests in a `while` loop), the resulting service graph looks
+similar to the following:
 
 <figure><img src="./img/servicegraph.png" alt="BookInfo Service Graph" title="BookInfo Service Graph" />
 <figcaption>BookInfo Service Graph</figcaption></figure>
 
 
 ## Verifying the installation
 
-1. Ensure the following Kubernetes services were deployed: "istio-pilot", "istio-mixer", "istio-ingress", "istio-egress",
-   "istio-ca", and, optionally, "grafana", "prometheus' and "servicegraph".
+1. Ensure the following Kubernetes services are deployed: `istio-pilot`, `istio-mixer`,
+   `istio-ingress`, `istio-egress`, and, optionally, `grafana`, `prometheus` and `servicegraph`.
```

Comment:
`istio-ca` is not shown in the output

Reply:
It should be

Reply:
Hmm okay. Whoever originally wrote this doc didn't get it in their output, and I didn't get it in mine either. perhaps I'm doing something wrong

Reply:
can you add istio-ca back? This is a bug and we can fix it. Or may be we can fix it later, when someone updates the output.